docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,274 @@
+    "estimator.options.dynamical_decoupling.enable = True\n",
+    "estimator.options.dynamical_decoupling.sequence_type = \"XpXm\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "56ed5785-e3a8-4dc5-8b55-ef876fccd592",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Pauli twirling\n",
+    "\n",
+    "Twirling, also known as [randomized compiling](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.94.052325), is a widely used technique for converting arbitrary noise channels into noise channels with more specific structure.\n",
+    "\n",
+    "Pauli twirling is a special kind of twirling that uses Pauli operations. It has the effect of transforming any quantum channel into a Pauli channel. Performed alone, it can mitigate coherent noise because coherent noise tends to accumulate quadratically with the number of operations, whereas Pauli noise accumulates linearly. Pauli twirling is often combined with other error mitigation techniques that work better with Pauli noise than with arbitrary noise.\n",
+    "\n",
+    "Pauli twirling is implemented by sandwiching a chosen set of gates with randomly chosen single-qubit Pauli gates in such a way that the ideal effect of the gate remains the same. The result is that a single circuit is replaced with a random ensemble of circuits, all with the same ideal effect. When sampling the circuit, samples are drawn from multiple random instances, rather than just a single one.\n",
+    "\n",
+    "![Depiction of Pauli twirling](/images/run/error-mitigation-explanation/pauli_twirling.png)\n",
+    "\n",
+    "Since most of the errors in current quantum hardware come from two-qubit gates, this technique is often applied exclusively to (native) two-qubit gates. The following diagram depicts some Pauli twirls for the CNOT and ECR gates. Every circuit within a row has the same ideal effect.\n",
+    "\n",
+    "![Depiction of gate twirls](/images/run/error-mitigation-explanation/gate_twirls.png)\n",
+    "\n",
+    "The Qiskit Runtime options for Pauli twirling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions). The following options are notable:\n",
+    "\n",
+    "- `enable_gates`: Set this to `True` to enable Pauli twirling of two-qubit gates.\n",
+    "- `num_randomizations`: The number of circuit instances to draw from the ensemble of twirled circuits.\n",
+    "- `shots_per_randomization`: The number of shots to sample from each circuit instance.\n",
+    "\n",
+    "The following code cell shows how to enable Pauli twirling and set these options for the estimator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c3e12d44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(backend=backend)\n",
+    "estimator.options.twirling.enable_gates = True\n",
```

Comment:
same with these options, it would be good to call out to the user which of these options you have to set vs which are optional

Reply:
Done.